oft-self-trace.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Good point. I completely overlooked that the nested project structure made this so complicated now. Maybe at some point in the far distant future, we need globs. :-)